Detectors/TPC/workflow/src/CATrackerSpec.cxx

```diff
@@ -72,10 +73,13 @@ DataProcessorSpec getCATrackerSpec(ca::Config const& specconfig, std::vector<int
   auto& processMC = specconfig.processMC;
   auto& caClusterer = specconfig.caClusterer;
   auto& zsDecoder = specconfig.zsDecoder;
+  auto& zsOnTheFly = specconfig.zsOnTheFly;
   if (specconfig.outputCAClusters && !specconfig.caClusterer) {
     throw std::runtime_error("inconsistent configuration: cluster output is only possible if CA clusterer is activated");
   }
+
   constexpr static size_t NSectors = o2::tpc::Sector::MAXSECTOR;
```

Comment:
You can actually get this from o2::tpc::constatnts, and if you want you can add the NEndpoints to that class.